doc/src/hexdoc/_templates/pages/hexcasting/brainsweep.html.jinja

```diff
@@ -2,11 +2,12 @@
 {% import "macros/recipes.html.jinja" as recipe_macros with context %}
 
 {% block body %}
-  {{ recipe_macros.generic(
+  {{ recipe_macros.render_mindcipe(page.recipe) }}
+  {# {{ recipe_macros.generic(
     [page.recipe],
     "name",
     _("hexdoc.pages.hexcasting.brainsweep.description"),
     _("hexdoc.pages.hexcasting.brainsweep.separator"),
-  ) }}
+  ) }} #}
```

Comment:
remove if not needed (also - this is the last place `generic()` is used, so you can remove that entire macro if you want)